spec/imports/imports.spec.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+var fs = require('fs-extra');
+var logger = require('log4js').getLogger('Plugins-e2e');
+var _ = require('lodash');
+var components = require('../../src/components');
```

Comment:
you can use `require('componts')` as we are modifying the NODE_PATH to support it. might need to merge from master..